src/frontend/apps/lti_site/components/VideoWidgetProvider/widgets/DownloadVideo/index.tsx

```diff
@@ -1,11 +1,16 @@
 import { Box, Button, Select, Text } from 'grommet';
-import React, { useMemo, useState } from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
 import { defineMessages, useIntl } from 'react-intl';
 import styled from 'styled-components';
 
 import { FoldableItem } from 'components/graphicals/FoldableItem';
+import { ToggleInput } from 'components/graphicals/ToggleInput';
 import { useCurrentVideo } from 'data/stores/useCurrentRessource/useCurrentVideo';
 import { videoSize } from 'types/tracks';
+import { useUpdateVideo } from 'data/queries';
+import { toast } from 'react-hot-toast';
+import { report } from 'utils/errors/report';
+import { Video } from 'types/tracks';
```

Comment:
Your order should be ordered and import fromn external libs should be in the fir import block (so you have to move `react-hot-toast` after `react`).